packages/webapp/src/components/HttpLabel.tsx

```diff
@@ -20,14 +21,28 @@ const styles = cva('', {
         }
     }
 });
+const sizesVerb = cva('text-[11px]', {
```

Comment:
in the docs we usually refers to http `methods` not `verbs`. https://docs.nango.dev/reference/integration-configuration

Would it make sense to also use `method` in the dashboard UI?

Reply:
good point 👌🏻